hera_pspec/uvpspec_utils.py

```diff
@@ -199,73 +210,164 @@ def select_common(uvp_list, spws=True, blpairs=True, times=True, pols=True,
 
-    if pols and len(common_pols) == 0:
-        raise ValueError("No polarizations were found that exist in all spectra.")
+    if polpairs and len(common_polpairs) == 0:
+        raise ValueError("No polarization-pairs were found that exist in all spectra.")
 
     # Apply selections
     out_list = []
     for i, uvp in enumerate(uvp_list):
-        _spws, _blpairs, _times, _lsts, _pols = None, None, None, None, None
+        _spws, _blpairs, _times, _lsts, _polpairs = None, None, None, None, None
 
         # Set indices of blpairs, times, and pols to keep
         if blpairs: _blpairs = common_blpairs
         if times: _times = common_times
         if lsts: _lsts = common_lsts
-        if pols: _pols = common_pols
+        if polpairs: _pols = common_polpairs
         if spws: _spws = [uvp.get_spw_ranges().index(j) for j in common_spws]
 
         _uvp = uvp.select(spws=_spws, blpairs=_blpairs, times=_times,
-                          pols=_pols, lsts=_lsts, inplace=inplace)
+                          polpairs=_polpairs, lsts=_lsts, inplace=inplace)
         if not inplace: out_list.append(_uvp)
 
     # Return if not inplace
     if not inplace: return out_list
 
 
+def polpair_int2tuple(polpair, pol_strings=False):
+    """
+    Convert a pol-pair integer into a tuple pair of polarization 
+    integers. See polpair_tuple2int for more details.
+    
+    Parameters
+    ----------
+    polpair : int or list of int
+        Integer representation of polarization pair.
+    
+    pol_strings : bool, optional
+        If True, return polarization pair tuples with polarization strings. 
+        Otherwise, use polarization integers. Default: True.
+    
+    Returns
+    -------
+    polpair : tuple, length 2
+        A length-2 tuple containing a pair of polarization 
+        integers, e.g. (-5, -5).
+    """
+    # Recursive evaluation
+    if isinstance(polpair, (list, np.ndarray)):
+        return [polpair_int2tuple(p, pol_strings=pol_strings) for p in polpair]
+    
+    # Check for integer type
+    assert isinstance(polpair, (int, np.integer)), \
+        "polpair must be integer: %s" % type(polpair)
+    
+    # Split into pol1 and pol2 integers
+    pol1 = int(str(polpair)[:-2]) - 10
+    pol2 = int(str(polpair)[-2:]) - 10
+    
+    # Check that pol1 and pol2 are in the allowed range (-8, 4)
+    if (pol1 < -8 or pol1 > 4) or (pol2 < -8 or pol2 > 4):
+        raise ValueError("polpair integer evaluates to an invalid "
+                         "polarization pair: (%d, %d)" 
+                         % (pol1, pol2))
+    # Convert to strings if requested
+    if pol_strings:
+        return (polnum2str(pol1), polnum2str(pol2))
+    else:
+        return (pol1, pol2)
+    
+
+def polpair_tuple2int(polpair):
+    """
+    Convert a tuple pair of polarization strings/integers into 
+    an pol-pair integer.
+    
+    The polpair integer is formed by adding 10 to each standardized 
```

Comment:
Seems like we might want to make this `adding 20 to each standardized polarization integer and then concatenating them`, because the dipole polarizations are `-5, -6, -7, -8`, which means that in the former case `XX,XX` would be `55`, which wouldn't follow a standard convention of a len-4 number, but in the latter case would be `1515`. LMK what you think.

We did a similar thing to `blpair` integers recently to make this more standardized.

Reply:
I should have pointed out that there is a leading zero, so `('xx', 'xx')` -> `505` (unless you think it should always be a len-4 number).

Reply:
I see. Yeah I guess its not a big deal because there is still a 1-to-1 mapping, so its up to you. For whatever reason, I find the integers themselves more readable when there is a set number of integers, like `1515` over `505`, but b/c you provide convenience functions for mapping them to strings its not a big deal to me!

Reply:
You've convinced me! I'm adding 20 now, so they will all be len-4 integers.